api/envoy/extensions/transport_sockets/tls/v3/tls_spiffe_validator_config.proto

```diff
@@ -57,4 +79,9 @@ message SPIFFECertValidatorConfig {
 
   // This field specifies trust domains used for validating incoming X.509-SVID(s).
   repeated TrustDomain trust_domains = 1 [(validate.rules).repeated = {min_items: 1}];
+
+  // This field specifies a trust domain mapping as a json object. Mutually
+  // excluse with trust_domains.
+  config.core.v3.DataSource trust_bundle_map = 2;
+
```

Comment:
How about just add a format enum/flag to the TrustDomain rather than adding this new `trust_bundle_map`?

/wait-any

Reply:
The SPIFFE bundle map can have many trust domains.  An enum for TrustDomain would only apply to a single one.  This is WIP and currently only reads in a single bundle, going to change it to support a map.

[Example of a map](https://docs.google.com/document/d/13KHycYIfRC-g42aEfo4_4inF_WauCXOBgZAcUGuxdgs/edit#heading=h.o2sg9lu1e74v)